aiidalab_widgets_base/computational_resources.py

```diff
@@ -207,10 +207,10 @@ def _validate_value(self, change):
             return None
 
         try:
-            _ = orm.load_code(code_uuid)
-        except NotExistent:
-            self.output.value = f"""The supplied code UUID '<span style="color:red">{code_uuid}</span>'
-                was not found in the AiiDA database."""
+            _ = UUID(code_uuid, version=4)
+        except ValueError:
+            self.output.value = f"""'<span style="color:red">{code_uuid}</span>'
+            is not a valid UUID."""
```

Comment:
I think the standard behavior is to raise a `TraitError` in this case, see [here](https://traitlets.readthedocs.io/en/stable/using_traitlets.html#validation-and-coercion).

Reply:
No, this is the exception raised when the `code_uuid` is not a valid UUID. `uuid.UUID` module will raise a `ValueError`
For example:
```python
from uuid import UUID

UUID("IAMNOTUUID", version=4)
```

Reply:
Yes, but **we** should raise a `TraitError` in response.